### PR TITLE
Adding documentation for installing Innovation with HA

### PR DESCRIPTION
**Pull request by cortex-le**
Information about installing and checking all the different components needed for HA with public and self signed certificates.

### CODE REVIEW

File: content/en/docs/getting-started/on-premise/multiple-server-with-ha.md [hunk 1]
```diff
@@ -88,40 +105,554 @@ TODO
...
+| Cortex.WindowsSmbRemoteRegistry | The ports used by Windows SMB and Remote Registry service. | 135, 137, 138, 139, 445 | TCP | Inbound |
+| Cortex.ServiceFabric.Customer1.ClusterConnectionEndpointPort | The port used by the client to connect to the cluster when client APIs are used. | 9000 | TCP | Inbound |
+| Cortex.ServiceFabric.Customer1.ClientConnectionEndpointPort | The port where the nodes communicate with each other. | 9001 | TCP | Inbound |
+| Cortex.ServiceFabric.Customer1.ServiceConnectionEndpointPort | The port used by the applications and services deployed on a node to communicate with the Service Fabric client on that particular node. | 9003 | TCP | Inbound |
+
+##### Microsoft Service Fabric Firewall Rules (present on all Application Servers, with Domain, Public and Private Profiles)
+
+These rules will all appear in Windows Firewall with names starting with ‘{CustomerName}.{NodeName} WindowsFabric’.
+
+|Name in Rule | Name in Config | Description | Default Port(s) | Protocol(s) | Direction | Program|
```
Comment (cortex-tp) [inline content/en/docs/getting-started/on-premise/multiple-server-with-ha.md:219]: Descriptions column width needs increasing

File: content/en/docs/getting-started/on-premise/multiple-server-with-ha.md [hunk 2]
```diff
@@ -88,40 +105,554 @@ TODO
...
+    * SQL Server 2019 Installation Guide for Cortex
+    * SQL Server 2016 Installation Guide for Cortex
+    * SQL Server 2016 Express Installation Guide for Cortex
+1. Copy the following artefact (the version number may differ) to the machine that you will be installing Cortex Gateway on:
+    * Cortex Evolution - Innovation 2022-RC.2022.1.2 - Gateway.zip
+1. Use the "Cortex Installation Guide" to install Cortex Gateway, beginning at the section "Cortex Gateway Installation". Ensure that you refer to and follow the instructions for installing prerequisites in the "Note" sections. Use the following points as they will differ from a Cortex Integrity installation:
+    * In the "Required Components", ensure that the prerequisites in the "Note" sections are followed ("System requirements" and "Install Internet Information Services") but do not install the additional Cortex components mentioed. For Cortex Innovation; only SQL Server and Cortex Gateway are needed, along with the other components detailed in this guide.
+    * Wherever "Cortex Gateway.zip" is mentioned, use the "Cortex Evolution - Innovation * - Block Packages.zip" file that was copied in the previous step.
```
Comment (cortex-tp) [inline content/en/docs/getting-started/on-premise/multiple-server-with-ha.md:476]: Wrong zip file specified

File: content/en/docs/getting-started/on-premise/multiple-server-with-ha.md (hunk 1 above)
Comment (cortex-le) [reply content/en/docs/getting-started/on-premise/multiple-server-with-ha.md]: got moved out into a new file and reformatted

File: content/en/docs/getting-started/on-premise/multiple-server-with-ha.md (hunk 2 above)
Comment (cortex-le) [reply content/en/docs/getting-started/on-premise/multiple-server-with-ha.md]: Fixed

**Pull request merged**